src/app/Main/Body/QueryPage/Results/Charts/BarChartResults.component.tsx

```diff
@@ -10,6 +10,7 @@ export function BarChartResultsComponent(props: {
     xAxisLabel: string;
     yAxisLabel: string;
     xAxisMax?: number;
+    isStackedBarChart: boolean;

```

Comment:
Consider using something else then `is` here. The variable is used as a setter from the outside, not a getter. So something like: `useStackedMode` or `displayAsStacked` or just `stacked` would be better.